components/menu/types/index.d.ts

```diff
@@ -69,6 +77,7 @@ export interface MenuItemProps {
      * When true, nested menu items are shown in a Popper
      */
     showSubMenu?: boolean
+    tabIndex?: number
```

Comment:
i think suffix (from a previous pr is missing here as well, can we add it as part of this pr?

Reply:
Will do